yarn.lock

```diff
@@ -0,0 +1,15566 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Could we delete the `yarn.lock` and add it to the `.gitignore` as we are using `pnpm`?